edgar-app/src/components/dashboardPages/home/HomeTreatmentsCard.tsx

```diff
@@ -0,0 +1,167 @@
+		>
+			<Box w="100%">
+				<Link href="/dashboard/chat">
+					<Button w="100%" size="customLg" id="edgar-dashboardHomePage-chat-button">
+						Consulter mes traitements
+					</Button>
+				</Link>
+			</Box>
+			<Box w="100%" h="2px" bg="blue.700" />
+			{treatments.length > 0 ? (
+				<>
+					<VStack spacing="8px" w="100%" align="start">
+						<Text size="boldMd">Vos médicaments d'aujourd'hui</Text>
+						<Box w="100%" h="2px" bg="blue.100" />
+						<VStack spacing="4px" align="start">
+							<Text size="boldLg">Matin</Text>
+							{treatments.map((treatment) => (
+								<Checkbox
+									key={treatment.id}
+									value={treatment.name}
+									onChange={() => handleChange(treatment.name)}
+									borderColor="blue.200"
+								>
+									<Text
+										size="boldMd"
+										textDecoration={
+											checkedItems.includes(treatment.name) ? 'line-through' : undefined
+										}
+									>
+										{treatment.number} x {treatment.name}
+									</Text>
+								</Checkbox>
+							))}
+						</VStack>
+						<Box w="100%" h="2px" bg="blue.100" />
+						<VStack spacing="4px" align="start">
+							<Text size="boldLg">Midi</Text>
+							{treatments.map((treatment) => (
+								<Checkbox
+									key={treatment.id}
+									value={treatment.name}
+									onChange={() => handleChange(treatment.name)}
+									borderColor="blue.200"
+								>
+									<Text
+										size="boldMd"
+										textDecoration={
+											checkedItems.includes(treatment.name) ? 'line-through' : undefined
+										}
+									>
+										{treatment.number} x {treatment.name}
+									</Text>
+								</Checkbox>
+							))}
+						</VStack>
+						<Box w="100%" h="2px" bg="blue.100" />
+						<VStack spacing="4px" align="start">
+							<Text size="boldLg">Soir</Text>
+							{treatments.map((treatment) => (
+								<Checkbox
+									key={treatment.id}
+									value={treatment.name}
+									onChange={() => handleChange(treatment.name)}
+									borderColor="blue.200"
+								>
+									<Text
+										size="boldMd"
+										textDecoration={
+											checkedItems.includes(treatment.name) ? 'line-through' : undefined
+										}
+									>
+										{treatment.number} x {treatment.name}
+									</Text>
+								</Checkbox>
+							))}
+						</VStack>
+						<Box w="100%" h="2px" bg="blue.100" />
+						<VStack spacing="4px" align="start">
+							<Text size="boldLg">Nuit</Text>
+							{treatments.map((treatment) => (
+								<Checkbox
+									key={treatment.id}
+									value={treatment.name}
+									onChange={() => handleChange(treatment.name)}
+									borderColor="blue.200"
+								>
+									<Text
+										size="boldMd"
+										textDecoration={
+											checkedItems.includes(treatment.name) ? 'line-through' : undefined
+										}
+									>
+										{treatment.number} x {treatment.name}
+									</Text>
+								</Checkbox>
+							))}
+						</VStack>
+					</VStack>
```

Comment:
You need to handle the data in another way, the backend will send you a list of treatments checked and you need to get the medical info of the patient and for each treatment you need to group and to display the treatment in the good order